pkg/apply/desiredset_compare.go

```diff
@@ -76,6 +79,8 @@ func prepareObjectForCreate(gvk schema.GroupVersionKind, obj runtime.Object) (ru
 	return obj, nil
 }
 
+// originalAndModified returns the original bytes from the oldObject's
+// "applied" annotation and the 'modified' bytes for the newObject
```

Comment:
nit: I don't think modified needs quotes
```suggestion
// originalAndModified returns the original bytes from the oldObject's
// "applied" annotation and the modified bytes for the newObject.
```